src/OrchardCore/OrchardCore.Abstractions/Localization/LocalizationCookieName.cs

```diff
@@ -1,19 +1,6 @@
-using Microsoft.AspNetCore.Localization;
-
 namespace OrchardCore.Localization;
 
-/// <summary>
-/// Contains a localization cookie names.
-/// </summary>
-public class LocalizationCookieName
+public static class LocalizationCookieName
 {
-    /// <summary>
-    /// Gets the cookie name that used for the admin site.
-    /// </summary>
-    public static readonly string AdminSite = ".OrchardCore.AdminSiteCulture";
-
-    /// <summary>
-    /// Gets the cookie name that used for the site.
-    /// </summary>
-    public static readonly string Site = CookieRequestCultureProvider.DefaultCookieName;
+    public const string AdminCulturePrefix = "admin_culture_";
```

Comment:
Seems you are pushing to use cookie prefix, still I'm wondering what's the issue of controlling the cookie path instead?!!

Reply:
I think the only limitation with setting the path is when you working with default tenant, coz `path=/` will be applied for the entire domain

Reply:
Yes and that's why we also need to differentiate by name, notice that a non default tenant may have an empty url prefix. But yes the cookie having the path "/" will appear under all tenants because e.g. `/tenant1` is a subpath of `/`.

We could only differentiate by path if all tenants has a non empty url prefix, or another solution would be to use this pattern for the path `/urlPrefix/adminPrefix` so that e.g. `/tenant1/admin` would not be considered as a subpath of `/admin`.

But that's okay to also differentiate by name as we already do for the auth and the AFT cookies.

Reply:
Seems my comment was outdated ;) I just come up with solution by appending `/Admin` in the path, test almost cases, everything works as expected. Please check the update PR if you have time

Reply:
See my last comment in your PR to see why using the shell name is not right.

Then yes just saw that you added `/admin` but in an harcoded way without using adminOptions, so that's wrong too.